src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -337,7 +335,7 @@ private void PostProcessProjectStatus(IEnumerable<ProjectData> projects)
 
         private void AddAnalyzerOutputFilePath(ProjectInfo project, ProjectData projectData)
         {
-            var property = project.AnalysisSettings.FirstOrDefault(p => p.Id.EndsWith(".analyzer.projectOutPath"));
+            var property = project.AnalysisSettings.FirstOrDefault(p => p.Id.EndsWith(".analyzer.projectOutPaths"));
```

Comment:
I would summon @duncanp-sonar on this one to be honest, because i don't know.

Reply:
So yes, i did a refactoring to better reflect multi-valued property, these 2 methods now have the same implementation.

Reply:
@mickael-caro-sonarsource I don't have enough context to comment, I'm afraid. Did you change the method again in your latest commit?

Reply:
Yep, in fact, as per this test on sonar-dotnet (https://github.com/SonarSource/sonar-dotnet/blob/fbdbf89fa8327d5da1cf2a6ec10dbd508d313622/sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/AbstractGlobalProtobufFileProcessorTest.java#L91) we are expecteing a comma-separated list of values, so that should be ok.

Reply:
@pavel-mikula-sonarsource i re-read your first comment, so yes, the initial version added the "whole" value simply because it was the single-valued property (without the "s") so that makes sense to modify it with a foreach to add those multiple values.